src/main/java/org/folio/dew/batch/bulkedit/jobs/processidentifiers/ItemFetcher.java

```diff
@@ -107,15 +110,9 @@ public synchronized ExtendedItemCollection process(ItemIdentifier itemIdentifier
               );
               extendedItemCollection.setTotalRecords(extendedItemCollection.getTotalRecords() + itemCollection.getTotalRecords());
             } catch (Exception e) {
-              if (e instanceof FeignException && ((FeignException) e).status() == 401) {
```

Comment:
Do we really need to remove logic related to handling affiliation errors? Especially, in logic related to Okapi.

Reply:
Where were reported issues for affiliation checking, in the same time the such issues were not observed while checking affiliations using consortia functionality.